components/blitz/src/ome/services/blitz/repo/ManagedImportRequestI.java

```diff
@@ -661,7 +661,10 @@ public MessageDigest parseData(
                 store.setChannelGlobalMinMax(
                         c, minMax[0], minMax[1], series);
             }
-            return null;
+            if (pixelType != FormatTools.FLOAT &&
```

Comment:
I'm still not understanding this. It seems like this doesn't calculate in *fewer* cases as opposed to *more* cases. If there were an extra `return null`, say after the if, then it would skip more cases, right?